projectForm.php

```diff
@@ -16,19 +34,47 @@
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
         <h1>Create a New Project</h1>
-        <form method="POST" action="editProject.php">
+        <form method="POST" action="<?php if(!empty($_GET)) { echo $editProject . '?id=' . $project['id']; } else { echo $addProject; }; ?>">
```

Comment:
you could do this by having one file with an if to check if there is an id sent

Reply:
it feels nicer to split it out? But I am planning on refactoring to a class in order to get stuff lookin better - lots of repeat codeeeeee